_maps/RandomRuins/SpaceRuins/spacehotel.dmm

```diff
@@ -1659,6 +1663,7 @@
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on{
 	dir = 1
 	},
+/obj/structure/mirror/magic/lesser,
```

Comment:
the magic mirror shouldnt be added due to the custom ghost role pr that makes this just redundant

Reply:
this was made before the pr so thats why it is there, will fix